components/widgets/menu/menu_items/restricted_indicator.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import FeatureRestrictedModal from 'components/feature_restricted_modal/feature_restricted_modal';
+import OverlayTrigger from 'components/overlay_trigger';
+import ToggleModalButton from 'components/toggle_modal_button';
+import Tooltip from 'components/tooltip';
+
+import {Constants, ModalIdentifiers} from 'utils/constants';
+
+import './restricted_indicator.scss';
+
+type RestrictedIndicatorProps = {
+    blocked?: boolean;
+    tooltipTitle?: string;
+    tooltipMessage?: string;
+    tooltipMessageBlocked?: string;
+    modalTitle?: string;
+    modalMessage?: string;
+    modalTitleAfterTrial?: string;
+    modalMessageAfterTrial?: string;
+}
+
+const RestrictedIndicator = ({
+    blocked,
+    tooltipTitle,
+    tooltipMessage,
+    tooltipMessageBlocked,
+    modalTitle,
+    modalMessage,
+    modalTitleAfterTrial,
+    modalMessageAfterTrial,
+}: RestrictedIndicatorProps) => {
+    const {formatMessage} = useIntl();
+
+    return (
+        <span className='RestrictedIndicator__icon-tooltip-container'>
+            <OverlayTrigger
+                delayShow={Constants.OVERLAY_TIME_DELAY}
+                delayHide={120000}
```

Comment:
Two minutes seems like a really long time. Should this be something more similar to `Constants.OVERLAY_TIME_DELAY`, e.g. 400 milliseconds?

Reply:
My bad, this was used only for debugging purposes, removing. Nice catch!